### PR TITLE
Governance edit

### PR DESCRIPTION
**Pull request by christopherdancy**
PR Intentions 
Issue #308 intention was to make gov details on create DAO editable. Included in this PR
1. proposalThreshold
2. quorum
3. executionDelay
4. lateQuorumExecution
5. voteStartDelay
6. votingPeriod
Are now all editable fields with accompanying helper text and recommendations.

image

Each field is editable and will result in a governor module being setup to the users specifications

### CODE REVIEW

**Comment by christopherdancy** [comment]
added easy to read recommend text
image

File: src/hooks/useDeployDAO.ts [hunk 1]
```diff
@@ -125,12 +123,12 @@ const useDeployDAO = ({
...
           abiCoder.encode(['address'], [addresses.timelockUpgradeable?.address]), // Timelock Impl
-          abiCoder.encode(['uint64'], [BigNumber.from('0')]), // vote extension
-          abiCoder.encode(['uint256'], [BigNumber.from('0')]), // Todo: change voteDelay back to 6545 blocks for prod
-          abiCoder.encode(['uint256'], [BigNumber.from('10')]), // Todo: change votingPeriod back to 45818 blocks (1 week)
-          abiCoder.encode(['uint256'], [BigNumber.from(proposalThreshold.toString())]), // Threshold
-          abiCoder.encode(['uint256'], [BigNumber.from(quorum.toString())]), // Quorum
-          abiCoder.encode(['uint256'], [BigNumber.from('1')]), // Access Control Index
+          abiCoder.encode(['uint64'], [BigNumber.from(lateQuorumExecution)]), // vote extension
+          abiCoder.encode(['uint256'], [BigNumber.from(voteStartDelay)]), // Todo: change voteDelay back to 6545 blocks (1 day) for prod
+          abiCoder.encode(['uint256'], [BigNumber.from(votingPeriod)]), // Todo: change votingPeriod back to 45818 blocks (1 week)
```
Comment (adamgall) [inline src/hooks/useDeployDAO.ts:128]: Small nitpick but can you please remove the `Todo:` comments from these lines?

**Review by adamgall — CHANGES_REQUESTED** [review]

**Comment by adamgall** [comment]
I'm able to enter non-numbers into these inputs, which seems wrong.

Screen Shot 2022-06-10 at 2 45 39 AM

**Review by adamgall — DISMISSED** [review]
I'm able to enter non-numbers into these inputs, which seems wrong.

173006838-decbf63b-94df-40f4-a342-ccd53fc53b46

**Comment by Da-Colon** [comment]
@christopherdancy can we split up some of the input boxes into sections with a title? something like this page? @dannydecent might could use your input on this. 
 
image

**Comment by Da-Colon** [comment]
Also we probably don't need the lock icon in the title any more
image

**Review by Da-Colon — COMMENTED** [review]
Other than these large amounts of props being passed into components. Everything seems to working well and looks great. I made a card to relook at this part of the code to see if we can reduce the amount of props, but not worried about messing with it now, as next round designs may change some of this anyway.

Functionally, everything seems to work well. Have you tested that these new values are taking affect as intended if changed? I have seeing the delayed vote working well. Has the other params been tested as well? @christopherdancy

**Review by Da-Colon — APPROVED** [review]
Card is being created for "Tokens Required" workflow on create proposal. but everything here LGTM #338

**Pull request merged**